staticaddr/deposit/fsm.go

```diff
@@ -44,13 +52,17 @@ var (
 
 // Events.
 var (
-	OnStart             = fsm.EventType("OnStart")
-	OnWithdrawInitiated = fsm.EventType("OnWithdrawInitiated")
-	OnWithdrawn         = fsm.EventType("OnWithdrawn")
-	OnExpiry            = fsm.EventType("OnExpiry")
-	OnExpiryPublished   = fsm.EventType("OnExpiryPublished")
-	OnExpirySwept       = fsm.EventType("OnExpirySwept")
-	OnRecover           = fsm.EventType("OnRecover")
+	OnStart              = fsm.EventType("OnStart")
```

Comment:
I propose to add godocs with short descriptions of what event is what to all elements of this var group.

Reply:
done, also for the states.